cypress/e2e/shared/labels.test.ts

```diff
@@ -23,7 +23,7 @@ describe('labels', () => {
     return `background-color: rgb(${red}, ${green}, ${blue});`
   }
 
-  it('can create a label', () => {
+  it.skip('can create a label', () => {
```

Comment:
should we skip the entire test instead of just this one? I would assume the remaining tests in this file will also fail

Reply:
As far as I can see it was only the `can create a label` test,  let me check if others are failing too.

Reply:
the flaky test fails with `Expected to find element: `[data-testid="label-card"]`, but never found it.`  I don't see others failing, you might be right

Reply:
We're going to let this CI run finish and see if it passes, if not I can skip the whole suite.